src/hacbs/components/TaskRunListView/TaskRunListView.tsx

```diff
@@ -20,6 +36,44 @@ const TaskRunListView: React.FC<Props> = ({ pipelineName, namespace }) => {
     },
   });
 
+  const [nameFilter, setNameFilter] = React.useState<string>('');
```

Comment:
Let's update the query param to be consistent with other pages.
```suggestion
 const [nameFilter, setNameFilter] = useSearchParam('name', '');
```

Use The `ComponentListView` as an example.